File/src/File.php

```diff
@@ -325,6 +325,10 @@ public function save($data = null)
         if ($data !== null) {
             $this->content($data);
         }
+        
+        if (is_link($this->filename)) {
+            $this->filename = realpath($this->filename);
+        }
 
         $filename = $this->filename;
```

Comment:
Can you move the fix after this line and just use `$filename`? Because of otherwise the URLs will not stay intact.

Reply:
sure. done.